Added documentations for metrics on the new home page and documentations for issues when creating database

hphamMolecula opened the pull request.

ljsinclair commented on docs/cloud/cloud-troubleshooting/issue-creating-database.md:
```diff
@@ -0,0 +1,49 @@
+nav_order: 5
+---
+
+# Creating database issues.
```
A question mark "?" is the only punctuation you can add to the end of a heading. And only when there's a question (e.g., "How do I <do something>?"

ljsinclair commented on docs/cloud/cloud-troubleshooting/issue-creating-database.md:
```diff
@@ -0,0 +1,49 @@
+
+Please contact us at [se@featurebase.com](mailto:se@featurebase.com) immediately to prevent loss of data and restore your sevice.
+
+## Issue - You have exhausted your trial credit balance.
```
You need a separate page for each An issue/cause/solution to make them easier to find.
Refer to other issues for guidance on the YAML page title (used in nav)
The first heading should be the specific issue being experienced

ljsinclair replied:
```
# Issue: cannot create databases

## Cause

This issue occurs when trial account credit has been used.

* [Learn more about FeatureBase trial accounts](https://www.featurebase.com/pricing){:target="_blank"}

## Solution

* [Contact FeatureBase support](https://www.featurebase.com/contact-us){:target="_blank"} to discuss an upgrade to a paid account.

```

ljsinclair commented on docs/cloud/cloud-troubleshooting/issue-creating-database.md:
```diff
@@ -0,0 +1,49 @@
+---
+title: Creating database issues
```
As mentioned below, each issue gets its own page which makes them easier to find in Nav and in search.

The YAML title should relate to the issue, e.g.,

```
title: Service stopped issue
```

ljsinclair commented on docs/cloud/cloud-troubleshooting/issue-creating-database.md:
```diff
@@ -0,0 +1,49 @@
+
+# Creating database issues.
+
+## Issue - Your service has been stopped.
```
New page

ljsinclair commented on docs/cloud/cloud-troubleshooting/issue-creating-database.md:
```diff
@@ -0,0 +1,49 @@
+
+Please upgrade to paid user or contact us at [se@featurebase.com](mailto:se@featurebase.com).
+
+## Issue - You have exhausted your credit balance.
```
New page

ljsinclair replied:
That's not the issue, that's the cause.

The issue is that the user can't create new databases

ljsinclair commented on docs/cloud/cloud-troubleshooting/issue-creating-database.md:
```diff
@@ -0,0 +1,49 @@
+
+### Cause
+
+You have ran out of your credit balance.
```
Grammar issue -- "ran" should be "run" because it's a past tense

"You have run out of credit."
"Your credit balance has run out."

gthrone25 commented:
@hphamMolecula Awesome job! Really appreciate you digging into docs and using includes! I've made a commit that changes a couple things:
1. Makes the metrics includes more modular so that they can be reused individually and with the dependency on title headings
2. Removed the image because maintaining them in a nightmare. I'm proving that by starting the necessary updates to our quick start guide. Note I can't finish this as dev/staging has issues with classic DBs, so we'll have to wait until that clears up
3. Added some additional info and context to metrics

Please take a look and let me know if it looks good to you. 

@ljsinclair in regards to your comments, I think we need some clarification. There is one issue "the creation of a database" but there are multiple potential causes (no credit, service stopped, etc.). Is that still multiple pages? I feel like it might be right as is but may just need multiple items in the Cause section? You added a couple pages recently with multiple causes: https://docs.featurebase.com/docs/community/com-troubleshooting/com-issue-linux-file-limits/. Please let us know

hphamMolecula commented:
I've added the fixes for 

1. grammar:
`"You have ran out of your credit balance." -> "Your credit balance has run out."`
2. removed period from headings
3. Added provided links to doc:
`* [Learn more about FeatureBase trial accounts](https://www.featurebase.com/pricing){:target="_blank"}

## Solution

* [Upgrade to paid account](https://docs.featurebase.com/docs/cloud/cloud-org/cloud-org-upgrade-to-paid/){target="_blank"}
`

Waiting for final decision regarding dividing the issues into multiple pages.
